src/pages/employee/absences/[id].tsx

```diff
@@ -0,0 +1,47 @@
+import EmployeeHeader from "@/components/EmployeeHeader";
+import MyHead from "@/components/MyHead";
+import AbsenceShow from "@/components/ShiftAbsenceShow";
```

Comment:
名前を変えてインポートすると後で混乱しやすいので、なるべく同じ名前でインポートすることをおすすめします。

```suggestion
import ShiftAbsenceShow from "@/components/ShiftAbsenceShow";
```

Reply:
https://github.com/tatata05/next-app/pull/11/commits/4e78a323539077b82fbd813014429d0889ff60a1
修正しました！